modules/gcp_bigquery_masked_dataset/main.tf

```diff
@@ -0,0 +1,66 @@
+resource "random_id" "random_id" {
+  byte_length = 4
+}
+
+resource "google_kms_key_ring" "google_kms_key_ring" {
+  name     = "${var.name}_${random_id.random_id.hex}"
+  location = var.location
+}
+
+resource "google_kms_crypto_key" "google_kms_crypto_key" {
+  #checkov:skip=CKV_GCP_82:Need to be able to delete because of integration test
+  name            = "${var.name}_${random_id.random_id.hex}"
```

Comment:
Typically we use hyphens in names for GCP resources, don't think we have that formaized, but know we do that outside of data platform.

Reply:
I think that depends on stuff, sometimes you can't use hyphens and must have `_` because of validation.

Reply:
will try though.